octoprint_discordremote/discord.py

```diff
@@ -37,20 +38,25 @@
 
 class Discord:
     def __init__(self):
-        self.channel_id = None  # enable dev mode on discord, right-click on the channel, copy ID
+        # enable dev mode on discord, right-click on the channel, copy ID (you can input multiple IDs separated with ",")
+        self.channels_id = None
         self.bot_token = None  # get from the bot page. must be a bot, not a discord app
         self.gateway_url = "https://discordapp.com/api/gateway"
-        self.postURL = None  # URL to post messages to, as the bot
+        # URL to post messages to, as the bot
+        self.postURL = "https://discordapp.com/api/channels/{}/messages"
         self.heartbeat_sent = 0
         self.heartbeat_interval = None
         self.last_sequence = None
         self.session_id = None
         self.web_socket = None  # WebSocket. Used for heartbeat.
         self.logger = logging  # Logger, uses default logging unless overridden
-        self.headers = None  # Object containing the headers to send messages with.
```

Comment:
I'd prefer if the PR had as little whitespace/formatting changes as possible, as it distracts from the overall change. Moving these comments is fine however, so leave them.